recipes/procps/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "procps" %}
+{% set version = "3.3.12" %}
+{% set sha256 = "b1036c109f271c7c50325b11a748236f8e58aa1dbafefb30c995ff1d05b4a1a8" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://gitlab.com/procps-ng/procps/repository/v3.3.12/archive.tar.gz
```

Comment:
You should replace the `v3.3.12` here with `v{{ version }}` so the version only needs to change in one place.